xbmc/video/dialogs/GUIDialogVideoManager.h

```diff
@@ -70,12 +61,23 @@ class CGUIDialogVideoManager : public CGUIDialog
   void RefreshSelectedVideoAsset();
   bool AddVideoAssetFilePicker();
 
+  struct ManagerMessages
+  {
+    // unused message id value 0 is used to detect messages not provided by derived classes
+    unsigned int ManagerTitleId{};
+    unsigned int RemoveTitleId{};
+    unsigned int RemoveTextId{};
+    unsigned int AddDialogTitleId{};
+    unsigned int AddNoDefaultMovieVersionId{};
```

Comment:
Please follow our coding guidelines. struct members must start with a small letter.

Reply:
yes I noticed right after pushing the commit...